DOCS-9255-mule-domain-limitation (#71)
mulesoft/docs-visualizer: modules/ROOT/pages/troubleshoot-visualizer.adoc

=== Why can't I see the source of traffic from Anypoint Connector for Database (Database Connector)?

When you use Database Connector, Anypoint Visualizer does not indicate the type of database, such as Postgres or Oracle, from which the traffic originates. Additionally, Anypoint Visualizer cannot differentiate between instances of a database.

> === Why can't I see metrics and connection information for some on-premises apps and APIs?

Author: ```suggestion
=== Why do some on-premises apps and APIs show incorrect or incomplete metrics and connection information?
```

> === Why do some on-premises apps and APIs show incorrect or incomplete metrics and connection information?

Reviewer: For any reason, on-premises apps sounds odd to me. I'd suggest something like:
== Why do some Mule apps and APIs deployed on premises show incorrect or incomplete metrics and connection information?

However, this might make the title too long.

Author: ```suggestion
=== Why do some Mule apps and APIs deployed on premises show incorrect or incomplete metrics and connection information?
```
